django_project/changes/models/sponsor.py

```diff
@@ -49,6 +49,28 @@ class Sponsor(models.Model):
         blank=False,
         unique=False)  # there is a unique together rule in meta class below
 
+    sponsorship_contact_name = models.CharField(
```

Comment:
we already have the sponsor contact person name here https://github.com/kartoza/projecta/pull/661/files#diff-371604757162da387f77bda08ad26881L58